### DISCUSSION TITLE
Relicensing

### DISCUSSION THREAD

**Discussion by data-man** [c3lang/c3c#302] [General]
Is it possible?
E.g. to Apache or MIT (or dual). I think it will increase the popularity of the nice project.
I noticed that you have changed the licence from BSD-2 to LGPL.
Just wondering: why?

**Comment by lerno** [comment]
The compiler itself is intended to be LGPL, while I intend to make the standard library MIT. This is because I don't want some company to fork the compiler and then not contribute back to the main compiler. But I also want to make the standard library totally easy to replace and amend for commercial platforms.

**Reply by data-man** [reply]
I'm no lawyer, but is it possible to use third-party libraries (MIT-licensed or Unlicensed) in the compiler?
I have lots of good candidates. :)

**Reply by lerno** [reply]
In the standard library, absolutely. The backend itself is LLVM, which is MIT if I recall correctly.

**Comment by lerno** [comment]
And the reason why the compiler is LGPL is because there's been a bit of malicious forks of other compilers that I wish to avoid.

**Reply by data-man** [reply]
Yes, I remember Zen.
